plugin-pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PinerySource.java

```diff
@@ -132,13 +132,21 @@ private Stream<PineryIUSForAnalysisValue> lanes(
                     lp.getLastModified() == null ? Instant.EPOCH : lp.getLastModified().toInstant();
                 addLane.accept(lp.getSequencerRunName(), lp.getLaneNumber());
 
+                Set<RunDtoContainer> containers = run.getContainers();
+                if (containers.size() > 1) {
+                  multipleContainers.labels(run.getName()).inc();
```

Comment:
The gauge is never lowered or reset, so we'd have to restart Shesmu to clear any alert. That's probably fine though since we're not expecting it to happen, and restarting Shesmu isn't a big deal either.

Reply:
If the gauge value isn't meaningful, it can stay like this, can't it?